### ISSUE TITLE
Rake uses depreacated tasks

### ISSUE THREAD

**Issue by stoune** [rtomayko/posix-spawn#31]
When running rake tasks you will receive a warning message:
```bash
$ rake test
rake/gempackagetask is deprecated.  Use rubygems/package_task instead
rake aborted!
cannot load such file -- rake/extensiontask

(See full trace by running task with --trace)
```

**Comment by rtomayko** [comment]
Fixed in #67.

**Comment by hramrach** [comment]
Please make it make sense:
 bundle install
If 'bundle' is not a typo you can use command-not-found to lookup the package that contains it, like this:
    cnf bundle
